CHANGELOG

```diff
@@ -2,6 +2,10 @@ Qtile x.xx.x, released XXXX-XX-XX:
     !!! Config breakage !!!
         - lazy.qtile.display_kb() no longer receives any arguments. If you passed it any arguments
           (which were ignored previously), remove them.
+        - The format argument for widget.Net() takes `down` and `down_suffix` instead of just `down`.
+          This facilitates formatting of `down` (the same for `up`). If you are using the `format parameter`,
+          consider setting the precision to avoid seeing a large number of decimals on the widget.
+          Eg: something like {down:.2f} for two decimal points.
```

Comment:
```suggestion
          This facilitates formatting of `down` (the same for `up`). If you are using the `format` parameter,
          consider setting the precision to avoid seeing a large number of decimals on the widget.
          Eg: `{down:.2f}` for two decimal points.
```